packages/variable/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mia-platform/variable",
-  "version": "6.2.0",
```

Comment:
Why these diffs are in this PR? Wasn't packagelock already updated in master?

Reply:
every time I work on a new feature I run the npx command lerna clear && bootstrap